python/tk_multi_publish2/summary_overlay.py

```diff
@@ -53,6 +53,20 @@ def __init__(self, parent):
         self.ui.info.clicked.connect(self.info_clicked.emit)
         self.ui.publish_again.clicked.connect(self.publish_again_clicked.emit)
 
+    def show_no_items_error(self):
+        """
+        Shows a special message when there is no items collected under an alternate
+        UI operation determined by the 'enable_manual_load' application option.
+        """
+        self.ui.icon.setPixmap(
+            QtGui.QPixmap(":/tk_multi_publish2/publish_failed.png")
+        )
+        self.ui.label.setText("<p>Could not find any items to publish.</p>")
```

Comment:
ok. i steered you wrong. your previous method looked better. i was hoping it wouldn't wrap it to 3 lines. let's go back to using `\n`

Reply:
That was actually my rationale for manually deciding where to break the line.
Done.